BlazarUI/app/scripts/components/repo-build/RepoBuildCancelButton.jsx

```diff
@@ -27,7 +28,10 @@ class CancelBuildButton extends Component {
     this.setState({
       cancelling: true,
       showModal: false
-    });    
+    });
+
+    const parentPath = window.location.pathname.slice(0, window.location.pathname.lastIndexOf('/'));
```

Comment:
@benrlodge any idea how to accomplish this without `window.location.pathname`? I'm trying to get back to the parent component - the branch build history page